src/query/v2/db_accessor.hpp

```diff
@@ -114,53 +113,49 @@ class VertexAccessor final {
 
   auto PrimaryKey(storage::v3::View view) const { return impl_.PrimaryKey(view); }
 
-  storage::v3::ShardOperationResult<bool> AddLabel(storage::v3::LabelId label) {
-    return impl_.AddLabelAndValidate(label);
-  }
+  storage::v3::ShardResult<bool> AddLabel(storage::v3::LabelId label) { return impl_.AddLabelAndValidate(label); }
```

Comment:
Maybe also remove these functions if it is easy. If it is too much hassle, please let's discuss what is the issue.

Reply:
These functions will be removed once we start implementing the query engine with multiframes.